projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/vite.config.ts

```diff
@@ -0,0 +1,26 @@
+import preact from '@preact/preset-vite';
```

Comment:
This file will no longer be needed after I'm done converting verbumBundleSize to webpack.config.js:

```
verbumBundleSize: spawnSync( 'gzip', [ '-c', 'dist/index.js' ] ).stdout.toString().trim()
			.length,
```